db/migrate/20210109000000_add_masto_cw_to_twitter_options.rb

```diff
@@ -0,0 +1,8 @@
+class AddMastoCwToTwitterOptions < ActiveRecord::Migration[5.1]
+  def change
+    execute <<-SQL
+      CREATE TYPE masto_cw_options AS ENUM ('CW_AND_CONTENT', 'CONTENT_ONLY', 'CW_ONLY');
+    SQL
+    add_column :users, :masto_cw_options, :masto_cw_options
```

Comment:
How do I express that the default value should be `CW_ONLY`? As it is, I expect this migration will leave this column at `NULL` by default, and then we need to handle the null case in the code. It would be nice if we could make this column non-nullable, with a default of `CW_ONLY`, so that we don't need to handle the null case in the application code.

Reply:
You can use it as if it were a string, like so:
`add_column :users, :masto_cw_options, :masto_cw_options, default: 'CW_ONLY'`